grpclb/src/main/java/io/grpc/grpclb/GrpclbLoadBalancer.java

```diff
@@ -649,24 +649,28 @@ public PickResult pickSubchannel(PickSubchannelArgs args) {
     final String token;
     final boolean isDrop;
 
+    private RoundRobinEntry(
+        PickResult result, GrpclbClientLoadRecorder loadRecorder, String token, boolean isDrop) {
+      this.result = checkNotNull(result);
+      this.loadRecorder = checkNotNull(loadRecorder, "loadRecorder");
+      this.token = checkNotNull(token, "token");
+      this.isDrop = isDrop;
```

Comment:
Good idea. I dropped the `isDrop` field and rely on `result == DROP_PICK_RESULT` for checking a drop entry.